code/game/objects/structures/watercloset.dm

```diff
@@ -249,6 +249,10 @@
 			if(isliving(G))
 				var/mob/living/L = G
 				wash_mob(L)
+				if(ishuman(L))
+					var/mob/living/carbon/human/H = L
+					if((H.wear_suit && !(H.wear_suit.clothing_flags & SHOWEROKAY))|| (H.w_uniform && !(H.w_uniform.clothing_flags & SHOWEROKAY)) || (H.shoes && !(H.shoes.clothing_flags & SHOWEROKAY)) || (H.ears && !(H.ears.clothing_flags & SHOWEROKAY)) || (H.gloves && !(H.gloves.clothing_flags & SHOWEROKAY)) || (H.wear_mask && !(H.wear_mask.clothing_flags & SHOWEROKAY)) || (H.head && !(H.head.clothing_flags & SHOWEROKAY)))
```

Comment:
okay what the fuck is this shit, make it readable

Reply:
its just repeated for each type of clothing how can i make it more readable

Reply:
```
result &= H.wear_suit && !(H.wear_suit.clothing_flags & SHOWEROKAY)
result &= H.blah blah
if(result)
   stupid message
```

THEN IF YOU THINK ABOUT IT, YOU CAN TURN IT INTO A PROC SO YOU DON'T COPYPASTE THE SAME CODE THREE TIMES

Reply:
🤣